interfaces/builtin/custom_device.go

```diff
@@ -451,8 +462,33 @@ func (iface *customDeviceInterface) UDevConnectedPlug(spec *udev.Specification,
 	}
 
 	// Now write all the rules
-	for _, rule := range deviceRules {
-		spec.TagDevice(rule)
+	for deviceName, rule := range deviceRules {
+		if rule == placeholderRule {
```

Comment:
Could we reverse this check? Just to simplify indentation, it also makes it easier to read.
```suggestion
		if rule != placeholderRule {
		    spec.TagDevice(rule)
		    continue
		}
```

Reply:
Yes absolutely, I like this much better. Thanks!